CONTRIBUTING.md

```diff
@@ -26,6 +26,19 @@ For more on how 10up writes and manages code, check out our [10up Engineering Be
 
 The `develop` branch is the development branch which means it contains the next version to be released. `trunk` contains the latest released version.  Always work on the `develop` branch and open up PRs against `develop`.
 
+## Getting set up
+
+1. Clone the project `git@github.com:10up/10up-toolkit.git` somewhere.
+1. `cd` into the cloned repository
+1. Run `npm install` (make sure you're using npm >= 7)
```

Comment:
```suggestion
1. Run `npm install`
```

Any version of Node 16 should bring NPM 7. It's fine to leave it too, just thought it'd be best to make it more prominent.